test/extensions/filters/http/jwt_authn/filter_integration_test.cc

```diff
@@ -54,7 +54,7 @@ class HeaderToFilterStateFilterConfig : public Common::EmptyHttpFilterConfig {
                                      Server::Configuration::FactoryContext&) override {
     return [](Http::FilterChainFactoryCallbacks& callbacks) -> void {
       callbacks.addStreamDecoderFilter(
-          std::make_shared<HeaderToFilterStateFilter>("jwt_selector", "jwt_selector"));
+          std::make_shared<HeaderToFilterStateFilter>("jwt-selector", "jwt-selector"));
```

Comment:
Given that we decided to keep the default as allow, maybe revert all of these test changes that we don't need?